tools/packaging/boms/windows.json

```diff
@@ -3487,6 +3251,30 @@
     "Pattern": "zh-Hant/WindowsFormsIntegration.resources.dll",
     "FileType": "NonProduct"
   },
+  {
+    "Pattern": "_manifest/spdx_2.2/manifest.spdx.json",
```

Comment:
missing 
Disable SBOM signing for CI and add extra files for packaging tests (#19729)

Reply:
Cherry-picked the change